docker-compose.yml

```diff
@@ -58,8 +58,6 @@ services:
     ports:
       - "50000:50000"
       - "30000:30000"
-    volumes:
```

Comment:
Not critical: electrs could use a data volume as well so that it does not have to reindex the whole chain on a restart.